lib/main.dart

```diff
@@ -1,3 +1,4 @@
+// Carlo
```

Comment:
Carlo General Feedback:
Good README.ME.
Good commits.
Good naming convention and indentations.
Great app. Very user friendly, aesthetically pleasing and complete.
Possibly add more comments to explain the code to readers.